frontend/src/app/patients/personSchema.ts

```diff
@@ -110,10 +111,15 @@ export function isValidBirthdate(date: string | undefined) {
     return false;
   }
   if (parsedDate.year() < 1900) {
-    return false;
+    // TODO: translations
```

Comment:
Should probably do in this PR if possible

Reply:
Mhm, I’ll get w/ content about these messages & translations!

Reply:
Done in bb81aa10